packages/frontend/src/hooks/useMinimumBid.ts

```diff
@@ -0,0 +1,18 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { useCall } from '@usedapp/core'
+
+import { useDevconContract } from './contract'
+
+export function useMinimumBid() {
+  const devconContract = useDevconContract()
+  const { value, error } =
+    useCall(
+      devconContract && {
+        contract: devconContract,
+        method: 'reservePrice',
+        args: [],
+      }
+    ) ?? {}
+  const minimumBid = value ? value[0] : BigNumber.from(0)
+  return { minimumBid, error }
+}
```

Comment:
This hook is very similar to: `useMinimumIncrement`  and `useAuctionTime`. I'd extract a common `useDevconParam` hook that takes the `method` name to query as a param and returns a BigNumber. I'd then use this `useDevconParam` hook in `useMinimumBid` , `useMinimumIncrement` and `useAuctionTime` to deduplicate code.